src/common/ipc.ts

```diff
@@ -61,6 +61,9 @@ export interface IpcBroadcastParams<A extends any[] = any> {
 }
 
 export function broadcastIpc({ channel, frameId, frameOnly, webContentId, filter, args = [] }: IpcBroadcastParams) {
+  if (!webContents) {
+    return
```

Comment:
Why this is needed? This method intended to work only in main process and calls from renderer must not happen.

Reply:
Maybe rename to `broadcastToViews`? 
Or maybe `webContents = webContents || remote.webContents` if this make it work from renderer?

Reply:
Anyway, silently ignore call (even unexpected) from renderer doesn't sound right IMHO..

Reply:
I'll revert this change, out-of-scope for this PR.